test/controllers/trips_controller_test.rb

```diff
@@ -1,8 +1,36 @@
 require "test_helper"
 
 describe TripsController do
+
   describe "show" do
-    # Your tests go here
+    it "will get show for valid id" do
+
+      driver = Driver.create(name: "Marta", vin: "RF5J464C70D9C3KBT")
+      passenger = Passenger.create(name: "Mickey", phone_num: "2070772909")
+      trip = Trip.create(driver_id: driver.id, passenger_id: passenger.id, date: Date.today, rating: 4, cost: 234)
+
+      valid_trip_id = trip.id
+      get "/trips/#{valid_trip_id}"
```

Comment:
Why not use the path method?

```suggestion
      get trip_path(valid_trip_id)
```